### DISCUSSION TITLE
Inbox tab

### DISCUSSION THREAD

**Discussion by TheLastProject** [jonaswinkler/paperless-ng#549] [Archived FRs]
The home page shows "Documents in inbox: 0" on the right. But what the inbox is is just very unclear.

I think it would be very helpful if there was an Inbox tab on the left like such just showing what is considered to be in the inbox:
![image](https://user-images.githubusercontent.com/1885159/107985166-da821580-6fc9-11eb-8467-dac486df6539.png)

**Comment by jonaswinkler** [comment]
You can configure paperless to behave just like that.

* Define a new "Inbox" tag. Name it anything you like, and tick the "is inbox tag" checkbox. By doing so, it gets assigned to all new documents.
* Anything with that tag (or other "inbox" tags, for that matter) is considered to be in your inbox.
* Define a saved view that shows documents with inbox tags, sorted descending by date added, and have that visible in the sidebar.

**Reply by TheLastProject** [reply]
Hmm, I have to admit I can't get that to work. I created a tag named Inbox and maked it as "is inbox tag" and created this view:

![image](https://user-images.githubusercontent.com/1885159/107986527-8dec0980-6fcc-11eb-8ffa-d35eef9aa07e.png)

But it just shows absolutely everything for some reason.

**Reply by shamoon** [reply]
I think the "has tag" value should be the ID # of the Inbox tag, not the name. BTW theres an interface for managing saved views within the Paperless-ng UI without going to the Django admin (like your screenshot)

**Reply by TheLastProject** [reply]
Ah, that was it, yeah. Thank you!

Also thanks for telling me about that interface in the UI, I didn't notice you could create views because I saw no button in Settings. Turns out I had to filter as I wanted and clicks "Views" and then "Save as".

Yeah, this works. But I still am leaving my feature request here because it seems odd for it to not be there by default given how Statistics on the home screen explicitly mentions "Documents in inbox: "

**Reply by jonaswinkler** [reply]
Except for managing mail accounts and rules, there should be no reason to go to the admin for anything. The entire admin for the "saved filters" isn't exactly supposed to be used by anyone; it's simply there because the framework generates that automatically, it's free.

I'll probably hide that Inbox line in the statistics if no inbox tag is defined.
